relayer-engine/src/storage/storage.ts

```diff
@@ -312,7 +320,13 @@ export class Storage {
       const rawWorkflows = await Promise.all(
         activeWorkflowsIds.map(id => redis.hGetAll(id)),
       );
-      return rawWorkflows.map(raw => this.rawObjToWorkflow(raw));
+      const workflows = rawWorkflows.map(raw => this.rawObjToWorkflow(raw));
```

Comment:
Why are we doing this sorting? Could this lead to confusion where the response differs from the underlying queue? ex. if we're using this function to get workflows in the "pending" queue but the order we receive in the response is different than the order in the actual queue

If we decide to include this sorting, could you leave a brief explanation in a comment?